src/containers/DAO/pages/DAOPage/hooks/useAssets.ts

```diff
@@ -37,9 +37,14 @@ const useAssets = (dao?: zDAO): UseAssetsReturn => {
 		try {
 			if (cache[dao[cacheKey]]) {
 				const cached = cache[dao[cacheKey]];
-				setTotalUsd(cached.totalUsd);
-				setAssets(cached.assets);
-				setIsLoading(false);
+				/* Wrapped this in a setTimeout to force it
+					 to run after state is reset above - there
+					 are probably better solutions!	*/
+				setTimeout(() => {
```

Comment:
Strange that 'setTimeout' is triggering the change

Reply:
Yeah, something strange going on when accessing a value from cache. There's almost definitely a much better fix, but need to keep things moving for now 🤷